source/common/config/new_delta_subscription_state.cc

```diff
@@ -240,21 +222,18 @@ void NewDeltaSubscriptionState::handleGoodResponse(
                                           message.removed_resources());
   }
 
-  if (Runtime::runtimeFeatureEnabled(
-          "envoy.reloadable_features.delta_xds_subscription_state_tracking_fix")) {
-    const auto scoped_update = ttl_.scopedTtlUpdate();
-    if (requested_resource_state_.contains(Wildcard)) {
-      for (const auto& resource : message.resources()) {
+  const auto scoped_update = ttl_.scopedTtlUpdate();
```

Comment:
Sorry, made a mistake in the previous comment.
The scope needs to be kept to ensure `scoped_update` is released at the end end of this block.

Reply:
No problem! Fixed it already!